packages/block-editor/src/components/block-draggable/dropping-insertion-point.native.js

```diff
@@ -50,6 +52,12 @@ export default function DroppingInsertionPoint( {
 	} = useSelect( blockEditorStore );
 
 	const { blocksLayouts, findBlockLayoutByClientId } = useBlockListContext();
+	const insets = useSafeAreaInsets();
+	const { height } = useWindowDimensions();
+	const safeAreaOffset = insets.top + insets.bottom;
+	const maxHeight =
+		height -
+		( safeAreaOffset + styles[ 'dropping-insertion-point' ]?.height );
```

Comment:
Another option for not having to add the optional chaining operator (i.e. `?`) would be to add this style with the `height` property into the style mocks ([reference](https://github.com/WordPress/gutenberg/blob/trunk/test/native/__mocks__/styleMock.js)).

Reply:
Oh I didn't know that existed 😅 I'll add it there, thanks for the suggestion!

Reply:
Addressed in https://github.com/WordPress/gutenberg/pull/40658/commits/b14228247c127752351943b00a81fdfc21a7b996